.gitignore

```diff
@@ -70,3 +70,6 @@ yarn-error.log
 
 # Storybook build
 storybook-static/
+
+# jest
+.jest-test-results.json
```

Comment:
I'm going to remove the output as part of implementing #105, but this is fine for now.